### ISSUE TITLE
[Burn-P3+ Prometheus Bug]: Appears to be dropping fires that are large enough to make it through?

### ISSUE THREAD

**Issue by BadgerOnABike** [BurnP3/BurnP3PlusPrometheus#7]
### Contact Details

@badgeronabike

### What happened?

When I keep temp files I have all the shapes to test what the areas actually are from the shapefiles produced by Pandora there are large discrepancies.

### What component are you seeing the problem on?

R

### Approvals Process

- [ ] Testing For Issue
- [ ] Merge

**Comment by BadgerOnABike** [comment]
When I print the areas we can begin to see some strange behaviours.
```
Running Iteration: 1; Fire ID: 1
Warning 1: Value 2578096602.76000023 of field AREA of feature 0 not successfully written. Possibly due to too larger number with respect to field width
[1] 26 NA
ssim-task-step=1
ssim-task-status=Simulating -> Iteration is 2 - Timestep is 0
Running Iteration: 2; Fire ID: 1
Warning 1: Value 136058933462.320007 of field AREA of feature 0 not successfully written. Possibly due to too larger number with respect to field width
Warning 1: Value 243858604644.140015 of field AREA of feature 0 not successfully written. Possibly due to too larger number with respect to field width
Running Iteration: 2; Fire ID: 2
Warning 1: Value 64950345880.0299988 of field AREA of feature 0 not successfully written. Possibly due to too larger number with respect to field width
[1]   0 651
ssim-task-step=1
ssim-task-status=Simulating -> Iteration is 3 - Timestep is 0
Running Iteration: 3; Fire ID: 1
Warning 1: Value 15402138448.1800003 of field AREA of feature 0 not successfully written. Possibly due to too larger number with respect to field width
Running Iteration: 3; Fire ID: 2
Warning 1: Value 19447183492.9799995 of field AREA of feature 0 not successfully written. Possibly due to too larger number with respect to field width
Warning 1: Value 48230689077.4899979 of field AREA of feature 0 not successfully written. Possibly due to too larger number with respect to field width
Running Iteration: 3; Fire ID: 3
Warning 1: Value 2542487118.80999994 of field AREA of feature 0 not successfully written. Possibly due to too larger number with respect to field width
Running Iteration: 3; Fire ID: 4
Warning 1: Value 64559232701.9300003 of field AREA of feature 0 not successfully written. Possibly due to too larger number with respect to field width
Running Iteration: 3; Fire ID: 5
Warning 1: Value 9381772681.37999916 of field AREA of feature 0 not successfully written. Possibly due to too larger number with respect to field width
[1] 156   0  25 643  91
ssim-task-step=1
ssim-task-status=Simulating -> Iteration is 4 - Timestep is 0
Running Iteration: 4; Fire ID: 1
Warning 1: Value 1099034096.1400001 of field AREA of feature 0 not successfully written. Possibly due to too larger number with respect to field width
Warning 1: Value 4657177380.63000011 of field AREA of feature 0 not successfully written. Possibly due to too larger number with respect to field width
Running Iteration: 4; Fire ID: 2
Warning 1: Value 2716215226.38000011 of field AREA of feature 0 not successfully written. Possibly due to too larger number with respect to field width
Running Iteration: 4; Fire ID: 3
Warning 1: Value 26045949690.5800018 of field AREA of feature 0 not successfully written. Possibly due to too larger number with respect to field width
[1]   0  26 259
ssim-task-step=1
ssim-task-status=Simulating -> Iteration is 5 - Timestep is 0
Running Iteration: 5; Fire ID: 1
Warning 1: Value 4885573527.18999958 of field AREA of feature 0 not successfully written. Possibly due to too larger number with respect to field width
[1] 49 NA
ssim-task-step=1
Called from: eval(expr, p)
Browse[1]> n
debug at C:/Users/bremoore/SyncroSim/Packages/burnP3PlusPrometheus/growth-pandora.R#682: updateRunLog("\nBurn Summary:\n", sum(!is.na(burnAreas)), " fires burned.\n", 
    sum(burnAreas < minimumFireSize, na.rm = T), " fires discarded due to insufficient burn area.\n", 
    round(sum(burnAreas >= minimumFireSize, na.rm = T)/sum(!is.na(burnAreas)) * 
        100, 0), "% of simulated fires were above the minimum fire size.\n")
Browse[2]> burnAreas
 [1]  26  NA   0 651 156   0  25 643  91   0  26 259  49  NA
```

**Comment by shreeramsenthi** [comment]
Fixed as of v0.4.6. This was an issue with checking size before converting units to hectares. Thanks for catching this!
